docker/cloudsim_websocket/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Ubuntu 18.04 with nvidia-docker2 beta opengl support
+FROM ubuntu:18.04
+
+RUN export DEBIAN_FRONTEND=noninteractive \
+ && apt-get update \
+ && apt-get install -y \
+    tzdata \
+ && ln -fs /usr/share/zoneinfo/America/Los_Angeles /etc/localtime \
+ && dpkg-reconfigure --frontend noninteractive tzdata \
+ && apt-get clean
+
+RUN apt-get update -qq \
+ && apt-get install --no-install-recommends -y -qq \
+        gnupg2 \
+        lsb-release \
+        sudo \
+        wget \
+ && apt-get clean -qq
+
+# sdformat8-sdf conflicts with sdformat-sdf installed from gazebo
+# so we need to workaround this using a force overwrite
+# Do this before installing ign-gazebo
+RUN /bin/sh -c 'echo "deb [trusted=yes] http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list' \
+ && /bin/sh -c 'wget http://packages.osrfoundation.org/gazebo.key -O - | apt-key add -' \
+ && /bin/sh -c 'apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654'
+
+# install ign-blueprint
+RUN apt-get update \
+&&  apt-get install -y \
+    ignition-blueprint \
```

Comment:
Not sure this applies for the websocket server, but should this be using Ignition Dome?